adoptability.md

```diff
@@ -22,28 +22,30 @@ global security researchers and developers to build a better
 product. The DHIS2 DPG also publishes [CVEs, along with a clear
 process for reporting vulnerabilities](https://dhis2.org/security).
 
-**KEY RECOMMENDATION**: The existence of multiple, diverse vendors providing commercial
+**KEY RECOMMENDATION**: If possible, work with vendors that can show solid, public 
+evidence of their contributions to the project. The existence of multiple, diverse 
+vendors providing commercial
 support is practically a guarantee that the product is as stable and
-reliable as possible. If you are considering working with one of these
-vendors, you should find solid public evidence of their contributions
-to the product.
+reliable as possible, while looking through public reviews and the
+bug report database will give you additional insight into a product's
+stability and reliability.
 
 EXAMPLE: The digital ID DPG [MOSIP](https://www.mosip.io/) supports a
 thriving community of several dozen commercial partners, such as
 [systems integrators and biometric solution
 providers](https://www.mosip.io/resource-partners/mosip-integrations). 
 
-**KEY RECOMMENDATION**: Thorough product installation and initial
-configuration documentation indicates a product that's actually meant
-to be used. A documented Application Programming Interface (API) is
-often important to supporting the full range of the programmatic
+**KEY RECOMMENDATION**: Ensure the availability of documented Application 
+Programming Interfaces (APIs), which are necessary to the full range of the programmatic
 functionality you'll probably need, especially if you will work with
 data at scale (although a lack of published APIs isn't uncommon for
 early stage open source products, but if this is the case, they should
 be on the project's roadmap). Relatedly, be sure to get a detailed
 understanding the product's ability to import/export data, as you will
 surely need to work with legacy data at some point. Data portability
-also enables you to switch systems or vendors, should the need arise.
```

Comment:
I think it would be better to lead with the installation and initial configuration documentation, then mention API docs next, the order it was in originally.  That's the order in which one would need the different kinds of documentation.

Reply:
makes total sense. I just tried to list recs in the order found in the subsequent text but it reads very oddly here!